### PR TITLE
feat: reimplement default role manager

### PR DESCRIPTION
**Pull request by seriouszyx**
Fix: https://github.com/casbin/jcasbin/issues/261

Signed-off-by: Yixiang Zhao <seriouszyx@foxmail.com>

### CODE REVIEW

**Comment by hsluoyz** [comment]
@tangyang9464

File: src/main/java/org/casbin/jcasbin/util/LRUCache.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+package org.casbin.jcasbin.util;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * @author Yixiang Zhao (@seriouszyx)
+ **/
+public class LRUCache<K, V> {
```
Comment (tangyang9464) [inline src/main/java/org/casbin/jcasbin/util/LRUCache.java:23]: Consider using LinkedHashMap directly to implement LRU cache

Comment (hsluoyz) [reply src/main/java/org/casbin/jcasbin/util/LRUCache.java:23]: @seriouszyx

**Comment by hsluoyz** [comment]
@tangyang9464

**Review by tangyang9464 — APPROVED** [review]
lgtm

**Pull request merged**